### PR TITLE
Rubocop

### PR DESCRIPTION
**Pull request by aitbali**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Metrics/LineLength:
+  Max: 120
+
+Style/Documentation:
+  Enabled: false
+
+inherit_gem:
+  relaxed-rubocop: .rubocop.yml
+
```
Comment (Showner) [inline .rubocop.yml]: it should inherit rubocop relaxed see https://relaxed.ruby.style/

Comment (Showner) [reply .rubocop.yml]: ive missed the relaxed rubocop gem, need extra time to check implementation

Comment (aitbali) [reply .rubocop.yml]: oui avec la gem on fait ca , ?

You could also get the styleguide via rubygems. Add to your Gemfile:

gem "relaxed-rubocop"
And in your .rubocop.yml:

inherit_gem:
  relaxed-rubocop: .rubocop.yml

Comment (Showner) [reply .rubocop.yml]: Alright, approved then

**Pull request merged**